docs/api.md

```diff
@@ -217,39 +217,51 @@ articles.sync()
 
 ### Synchronization strategies
 
-The `sync()` method accepts a `strategy` option, which itself accepts the following values:
+For publication conflicts, the `sync()` method accepts a `strategy` option, which itself accepts the following values:
 
-- `Collection.strategy.MANUAL` (default): Conflicts are reflected in a `conflicts` array as a result, and need to be resolved manually.
+- `Collection.strategy.MANUAL` (default): Conflicts are reflected in a `conflicts` array as a result, and need to be resolved manually;
 - `Collection.strategy.SERVER_WINS`: Server data will be preserved;
 - `Collection.strategy.CLIENT_WINS`: Client data will be preserved.
 
+> Note:
+> `strategy` only applies to *outgoing* conflicts. *Incoming* conflicts will still
+> be reported in the `conflicts` array. See [`resolving conflicts section`](#resolving-conflicts).
+
+
 You can override default options by passing `#sync()` a new `options` object; Kinto will merge these new values with the default ones:
 
 ```js
+import Collection from "kinto/lib/collection";
+
 articles.sync({
   strategy: Collection.strategy.CLIENT_WINS,
   headers: {Authorization: "Basic bWF0Og=="}
 })
-  .then(console.log.bind(console));
+  .then(result => {
+    console.log(result);
+  })
```

Comment:
Just to explicit that the *result* we're talking about is coming here.
I had the feeling that newcomers might not get always get along with JavaScript brevity. But I might be wrong, and would be willing to revert this change :)